runtime/onert/backend/train/BackendContext.cc

```diff
@@ -91,10 +117,8 @@ backend::train::ITensorRegistry *BackendContext::genTrainingTensors()
     // NOTE Assuming there is no layout changes (Always assume NHWC or UNKNOWN)
     assert(tgraph.layout() != ir::Layout::NCHW);
 
-    // TODO Different shape of back propagation tensor
-    ir::OperandInfo backend_info{obj.shape(), obj.typeInfo(), obj.info().memAllocType(),
-                                 obj.isConstant()};
-    tensor_builder->registerBackwardTensorInfo(ind, backend_info, ir::Layout::NHWC);
+    tensor_builder->registerBackwardTensorInfo(ind, createBackwardTensorInfo(obj),
```

Comment:
Could you give me an example of `BackwardTensor`? 

In my understanding, 
* `DisposableBackwardTensor`: Tensor for backpropagation. Once propagation is done, It can be freed. 
* `BackwardTensor`: Used for backpropagation but can not be freed..? I couldn't think of a concrete example of this tensor.